tools/RAiDER/cli/__init__.py

```diff
@@ -36,7 +36,7 @@ class AttributeDict(dict):
             raster_format='GTiff',
             file_format='GTiff',
             download_only=False,
-            output_directory=os.getcwd(),
```

Comment:
Is this needed? os.getcwd() version is OS agnostic whereas I think '.' only works on *nix systems.

Reply:
I'm pretty sure pytest wouldn't allow it, but I will verify.